lib/developer_portal/app/controllers/developer_portal/admin/account/account_plans_controller.rb

```diff
@@ -11,7 +11,7 @@ class DeveloperPortal::Admin::Account::AccountPlansController < ::DeveloperPorta
     liquify prefix: 'account_plans', only: :index
 
     def index
-      account_plans = Liquid::Drops::AccountPlan.wrap(site_account.account_plans.published)
+      account_plans = Liquid::Drops::AccountPlan.wrap(site_account.account_plans.published.ordered)
```

Comment:
This change would probably fix the tests without the need to update the [JavaScript](https://github.com/3scale/porta/pull/3784/files#r1600683948), because the behavior would be the same as previously - the plans would appear in the order of creation, so the current plan (Default) would be first in the list, and hence it will be the one selected. And on selecting another plan the button would appear.